src/assets/styles/toolbar.scss

```diff
@@ -1,16 +1,75 @@
 .toolbar {
   text-align: center;
   background: $dark;
-  padding: 1rem;
```

Comment:
A couple misc styling notes -- IMO we can circle back and do these things later, just noting them:

- Since we are doing responsive styling / targeting mobile -> desktop, probably will be easier if we do mobile-first, i.e., have the general styles that apply to all screens, then anything that is specific to larger screens wrapped in media queries

- In line with this, fixed pixel values and fixed dimensions for layout-y things can be pretty brittle, wherever possible if we can use ems / percentages / ways to express layout/positioning/sizing in relative terms, the more flexible it'll be across different screen sizes and less bugs to fix later